### DISCUSSION TITLE
Is there a way i could contribute by translating replay to my mother language (Croatian)

### DISCUSSION THREAD

**Discussion by Marki2019** [nyx-lyb3ra/Replay#106] [Q&A] [answered]
I'm not that good with Vala and i would like to contribute by translating replay.
Is there any way i could do that?

**Comment by Marki2019** [comment]
P.S. Some things cannot be translated very good, like the word "About" or "Replay".

**Comment by nyx-lyb3ra** [comment, marked as answer]
Uhm... I need to do some changes in order to you to be able to translate Replay (mainly add the files to be translated to POTFILES), but I strongly suggest you to wait at least until #66 got implemented ;)

**Comment by Marki2019** [comment]
kk :D

btw, I already translated stuff that was translatable.

**Comment by nyx-lyb3ra** [comment]
Then submit a draft pr :P

**Comment by Marki2019** [comment]
There, i submitted it.
